content/200-concepts/100-components/02-prisma-client/000-working-with-prismaclient/115-connection-pool.mdx

```diff
@@ -57,6 +57,115 @@ datasource db {
 }
 ```
 
+#### Viewing the connection pool size
+
+The number of connections Prisma Client uses can be viewed using [logging](/concepts/components/prisma-client/working-with-prismaclient/logging) and [metrics](/concepts/components/prisma-client/metrics).
+
+Using the `info` [logging level](/reference/api-reference/prisma-client-reference#log-levels)<span class="api"></span>, you can log the number of connections in a connection pool that are opened when Prisma Client is instantiated.
+
+For example, consider the following Prisma Client instance and invocation:
+
+<CodeWithResult expanded="{true}">
+<cmd>
+
+```ts
+import { PrismaClient } from '@prisma/client'
+
+const prisma = new PrismaClient({
+  log: ['info'],
+})
+
+async function main() {
+  await prisma.user.findMany()
+}
+
+main()
+```
+
+</cmd>
+<cmdResult>
+
+```text no-copy
+prisma:info Starting a postgresql pool with 21 connections.
+```
+
+</cmdResult>
+</CodeWithResult>
+
+When the `PrismaClient` class was instantiated, the logging notified `stdout` that a connection pool with 21 connections was started.
```

Comment:
Hmmmm. Documenting this kinda makes it a public API, which the logs _content_ usually is not.
I don't want people to build tooling on that and then be unhappy when we (accidentally or intentionally) change that.
Ideas how to handle?

Reply:
We could add a warning that the output of `log: ['info']` can change in any release?

Reply:
```suggestion
When the `PrismaClient` class was instantiated, the logging notified `stdout` that a connection pool with 21 connections was started.

<Admonition type="warning">

Note that the output generated by `log: ['info']` can change in any release without notice. Be aware of this in case you are relying on the output in your application or a tool that you're building.

</Admonition>
```